### PR TITLE
Link to achieved badge

### PR DESCRIPTION
**Pull request by gideonthomas**
Relies on https://github.com/mozilla/learning.mozilla.org/pull/2339 landing first

### CODE REVIEW

File: pages/badges/badge-single.jsx [hunk 1]
```diff
@@ -305,7 +310,12 @@ var BadgePage = React.createClass({
 
     return (
       <div className="badge-achieved">
-        <h3 className={'text-light'}>Congrats, you were awarded this badge!</h3>
+        <h3 className={'text-light'}>
+          Congrats, you were awarded this badge!
+        </h3>
+        <h4>
```
Comment (Pomax) [inline pages/badges/badge-single.jsx]: This should probably not be a heading but a `<p>`

**Comment by gideonthomas** [comment]
@Pomax re-r?

File: README.md [hunk 2]
```diff
@@ -123,6 +123,7 @@ software works.
 `LOGIN_EMULATION_USERNAME` | determines the username that is used when login emulation is on and login status is not set to logged out. defaults to 'testuser'.
 `FULL_SUBDOMAIN_FOR_COOKIE` | A domain string, used by Optimizely to determine cookie domain that is used by Optimizely. defaults to 'learning.mozilla.org'
 `ENABLE_PONTOON` | set this to `on` to inject the Pontoon client-side localization script.
+`CREDLY_BASE_URL` | The main url that you can access the Credly interface through.
```
Comment (Pomax) [inline README.md]: + "defaults to https://staging.credly.com" just to be explicit?

File: pages/badges/badge-single.jsx [hunk 3]
```diff
@@ -10,7 +10,11 @@ var React = require('react'),
     Divider = require('../../components/Divider.jsx'),
     BadgesAPI = require('../../lib/badges-api'),
     TeachAPI = require('../../lib/teach-api'),
-    Modal = require('../../components/modal.jsx');
+    Modal = require('../../components/modal.jsx'),
+
+    config = require('../../config/config');
+
+var ACHIEVED_BADGE_BASE_URL = config.CREDLY_BASE_URL.replace(/\/?$/, '/credit/');
```
Comment (Pomax) [inline pages/badges/badge-single.jsx]: I kind of feel like we should not be doing this here - we should ensure the conf var has the trailing `/`, or does not have a trailing `/`, in the config object, when we read in the var from the environment for use in the app.

**Comment by Pomax** [comment]
looks good to me. I'll file a general followup for making the use of our API vars uniform (i.e. they either all end in `/` or they don't)

**Pull request merged**